nuxeo-core/nuxeo-core-bulk/src/main/java/org/nuxeo/ecm/core/bulk/BulkActionDescriptor.java

```diff
@@ -24,17 +25,16 @@
 import org.nuxeo.common.xmap.annotation.XObject;
 
 /**
- * The Bulk Operation Service configuration.
- *
```

Comment:
No description at all?

Reply:
Not sure this is valuable, as it only contains the action name.